custom/icds_reports/utils/aggregation_helpers/agg_awc.py

```diff
@@ -308,6 +310,44 @@ def updates(self):
             'start_date': self.month_start
         }
 
+        yield """
```

Comment:
is there any way to do this without adding additional updates to agg awc. this is already a very slow part of the aggregation, and this is going to make it that even slower